### ISSUE TITLE
p. 155 그룹 애너그램 문제

### ISSUE THREAD

**Issue by vacuum74** [onlybooks/algorithm-interview#108]
리트 코드 문제들이 수시로 수정되나요?

이 문제는 출력 내용이 달라졌어요.
[["bat"],["nat","tan"],["ate","eat","tea"]]

그래서 return 값을
sorted(map(sorted,anagrams.values()), key = len)
sorted([sorted(val) for val in anagrams.values()], key = len)

두 가지 정도로 수정해서 해봤는데요.
더 좋은 코드가 있는지 궁금합니다.
또... list 표현식과 map이 어느 정도 차이가 있는지도 궁금하구요.

**Comment by likejazz** [comment]
네, 리트코드 테스트케이스는 수시로 수정되고 그래서 출간 시점과 내용이 달라질 수 있습니다. 변경된 내용들은 발견하는대로 이 곳 깃헙에 추가하도록 하겠습니다.
